core/internal/cltest/heavyweight/orm.go

```diff
@@ -20,41 +15,45 @@ import (
 
 	"github.com/jmoiron/sqlx"
 
-	"github.com/smartcontractkit/chainlink/v2/core/cmd"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils/configtest"
 	"github.com/smartcontractkit/chainlink/v2/core/services/chainlink"
 	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
 	"github.com/smartcontractkit/chainlink/v2/core/store/dialects"
 	"github.com/smartcontractkit/chainlink/v2/core/store/models"
+	"github.com/smartcontractkit/chainlink/v2/internal/testdb"
 )
 
 // FullTestDBV2 creates a pristine DB which runs in a separate database than the normal
 // unit tests, so you can do things like use other Postgres connection types with it.
 func FullTestDBV2(t testing.TB, overrideFn func(c *chainlink.Config, s *chainlink.Secrets)) (chainlink.GeneralConfig, *sqlx.DB) {
-	return prepareFullTestDBV2(t, false, true, overrideFn)
+	return KindFixtures.PrepareDB(t, overrideFn)
 }
 
 // FullTestDBNoFixturesV2 is the same as FullTestDB, but it does not load fixtures.
 func FullTestDBNoFixturesV2(t testing.TB, overrideFn func(c *chainlink.Config, s *chainlink.Secrets)) (chainlink.GeneralConfig, *sqlx.DB) {
-	return prepareFullTestDBV2(t, false, false, overrideFn)
+	return KindTemplate.PrepareDB(t, overrideFn)
 }
 
 // FullTestDBEmptyV2 creates an empty DB (without migrations).
 func FullTestDBEmptyV2(t testing.TB, overrideFn func(c *chainlink.Config, s *chainlink.Secrets)) (chainlink.GeneralConfig, *sqlx.DB) {
-	return prepareFullTestDBV2(t, true, false, overrideFn)
+	return KindEmpty.PrepareDB(t, overrideFn)
 }
 
 func generateName() string {
 	return strings.ReplaceAll(uuid.New().String(), "-", "")
 }
 
-func prepareFullTestDBV2(t testing.TB, empty bool, loadFixtures bool, overrideFn func(c *chainlink.Config, s *chainlink.Secrets)) (chainlink.GeneralConfig, *sqlx.DB) {
-	testutils.SkipShort(t, "FullTestDB")
+type Kind int
 
-	if empty && loadFixtures {
-		t.Fatal("could not load fixtures into an empty DB")
-	}
+const (
+	KindEmpty Kind = iota
+	KindTemplate
+	KindFixtures
```

Comment:
I always had a hard time reading these funcs, but never looked too closely before. The bool params were not only hard to read, but also mutually exclusive. IMHO it is simpler to combine them in to one option. Before combining, I had just moved them to a separate config struct, so here I ended up settling on a method instead of a func. I considered undoing that, but FWIW this way is nicer for callers outside the package, since it avoids repeating the long package name:
```go
cfg, db := heavyweight.KindEmpty.PrepareDB(t, nil)
```
vs.
```go
cfg, db := heavyweight.PrepareDB(t, heavyweight.KindEmpty, nil)
```